torch/_dynamo/variables/builtin.py

```diff
@@ -1120,8 +1120,7 @@ def call_getattr(
                     ):
                         tuple_args = [SourcelessBuilder()(tx, bases[0])]
                     else:
-                        unimplemented(f"unexpected sourceless type bases: {bases}")
-
+                        tuple_args = [ConstantVariable.create(b) for b in bases]
```

Comment:
Use SourcelessBuilder instead of ConstantVariable, it is more symmetric with above.